### PR TITLE
i56: widget tests for otherAuthOptionsPage

### LINKED ISSUE #56: Complete widget tests for OtherAuthOptionPage
**Actions**
- [x] Sign in with email chip dispatches correct action
- [x] Create an account chip dispatches correct action

- [x] Sign in button dispatches correct action
- [x] Create an account button dispatches correct action

- [x] Sign in with "provider default" dispatches correct action

text feilds
- [x] email feild dispatches..
- [x] password feild dispatches..
- [x] retype password feild dispatches..

 - back button dispatches navigation actions - just uses default appbar back button 

**State changes** 
- [x] show progress indicator when AuthStep.waitingForInput
- [x] show create account button and repeat password feild when EmailAuthmode = create account
- [x] show default ui  when EmailAuthmode = sign in

### PR DESCRIPTION
**Pull request by tamari-gray**
fixes #56

### CODE REVIEW

File: lib/widgets/auth/other_auth_options_page.dart [hunk 1]
```diff
@@ -59,15 +59,10 @@ class OtherAuthOptionsPage extends StatelessWidget {
                           RepeatPasswordTextField(
                             visible: vm.showPassword,
                           ),
+                        // SizedBox(height: 50), // commented out as hack to do widget test without overlay error
+                        if (vm.mode == EmailAuthMode.signIn) SignInButton(),
```
Comment (nickmeinhold) [inline lib/widgets/auth/other_auth_options_page.dart:63]: addressed with #178

File: lib/widgets/auth/other_auth_options_page.dart [hunk 2]
```diff
@@ -59,15 +59,10 @@ class OtherAuthOptionsPage extends StatelessWidget {
                           RepeatPasswordTextField(
                             visible: vm.showPassword,
                           ),
+                        // SizedBox(height: 50), // commented out as hack to do widget test without overlay error
```
Comment (nickmeinhold) [inline lib/widgets/auth/other_auth_options_page.dart:62]: addressed with #178

**Review by nickmeinhold — APPROVED** [review]
So good!  A PR full of amazing tests!  Thanks heaps!!

**Comment by tamari-gray** [comment]
hi @nickmeinhold,
just a reminder to delete the branch. (unless this branch was left intentionally)

**Comment by nickmeinhold** [comment]
@tamari-gray thanks so much!  I really wanna get this stuck in my memory - think I need some kind of mild punishment every time I forget... if you think of anything I'm all ears!

**Pull request merged**